lib/HttpSignatureStrategy.js

```diff
@@ -171,3 +178,21 @@ function _timestampBeforeNow(timestamp) {
   var tsDate = new Date(timestamp);
   return tsDate < now;
 }
+
+// TODO didio.get is returning undefined
+function getDidPublicKey(publicKey, callback) {
+  if(publicKey.substring(0, 4) === 'did:') {
+    /*didio.get(publicKey, function(err, doc) {
```

Comment:
I tried debugging did-io and I couldn't figure out why it was'nt working. The publicKey variable that's passed into didio.get() is 'did:4ddbace2-4a52-4a63-80b1-3885d556aced/keys/1'.